PSO-Application.md

```diff
@@ -0,0 +1,114 @@
+(now the members of the PlanetaryPy TC) to create something for
+planetary sciences that was akin to the Astropy project for astronomy.
+
+At the Planetary Data Workshop in June, 2019, we found that there
+was a real interest in the Planetary Sciences technical community
+for an active planetary Python resource. Many people pointed to the
+astropy project as a possible analog or goal.  The code currently
+within the GitHub PlanetaryPy organization is a great starting
+point, and the naming is right.
+
+The goal is to take that existing codebase, plus a solid governance
+model taken from the Planetary Software Organization, and eventually
+work to build something that is as robust as the Astropy Project.
+We are just beginning this process, and the intent is to slowly and
+methodically build towards a healthy Python ecosystem for planetary
+sciences.
+
+Participation in PlanetaryPy activities is completely voluntary.
+None of the PlanetaryPy TC members are currently funded to work on
+PlanetaryPy.
+
+
+## Metrics
+
+Since the repos currently under the GitHub planetarypy organization
+have always been a loose federation, no consistent or widespread
+user metrics are available.  However, we will use the planetarypy/pvl
+library as an example case.
+
+The `pvl` project, although it has had no commits in more than two
+years, is widely used by planetary science Python developers to
+deal with the pervasive parameter value language (PVL) standard in
+Planetary Data System (PDS) archives, and used in image labels
+written by the Integrated Software for Imagers and Spectrometers
+(ISIS) software.  It is officially listed as 'being used' by more
+than 20 other GitHub projects, and is likely used by many more.
+Its hosting on the Python Package Index (PyPI) indicates that it
+is downloaded 1.2 thousand times per month.
+
+Simple download counts are one metric, but the fact is that planetary 
+science software and analysis needs a `pvl` library as much as the
+Python standard library needs a `csv` and a `json` library.  And it is 
+the goal of the PlanetaryPy Project to provide libraries like `pvl`
+and other data processing libraries (like those provided by `astropy`)
+to the planetary sciences community.
+
+
+## Scope
+
+The PlanetaryPy TC will have sole responsibility and
+discretion over the Planetary Project in the following areas:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Governance process and practices.
+* Contribution process and practices.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators and Working Groups.
+
+
+## Governance, Contributions, Code of Conduct
+
+We plan to adopt the Governance model from the Planetary Software Organization.
+We plan to use the same Code of Conduct (based on the Contributor Covenant, version 1.4)
+that the PSO uses, and a Contribution model that is also similar.
```

Comment:
why are they not *exactly* the same for the contribution model? Summarize the changes here.

Reply:
Also a fine point, look for the next commit to the PR, which will address this.